Exercises/chapter-10/e2-roads-module/graph.js

```diff
@@ -0,0 +1,18 @@
+export default function buildGraph(edges) {
```

Comment:
I'd include this in the roads and mail route file. Too many small files is just as rough as impenetrably large ones because it's so much work to get everything you need on your screen and switch between them